app/src/test/java/com/example/android/psiak/FirebasePresenterTest.java

```diff
@@ -0,0 +1,113 @@
+package com.example.android.psiak;
+
+import com.example.android.psiak.Firebase.FirebaseActivityContract;
+import com.example.android.psiak.Firebase.FirebaseDataListener;
+import com.example.android.psiak.Firebase.FirebasePresenter;
+import com.example.android.psiak.Model.DogFirebase;
+import com.example.android.psiak.Repository.Repository;
+import com.google.firebase.database.DatabaseException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Created by serwis on 23.11.2017.
```

Comment:
Why don't we use JAVADOC instead?
`@author <a href="mailto:my_email@email.exmaple.com">Grzegorz Kwasniewski</a>`
or
`@author Grzegorz Kwasniewski`